exchange/bidder.go

```diff
@@ -58,7 +59,19 @@ type AdaptedBidder interface {
 	//
 	// Any errors will be user-facing in the API.
 	// Error messages should help publishers understand what might account for "bad" bids.
-	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestOptions bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor, ruleToAdjustments openrtb_ext.AdjustmentsByDealID) ([]*entities.PbsOrtbSeatBid, []error)
+	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, auctionOptions *auctionOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor, ruleToAdjustments openrtb_ext.AdjustmentsByDealID) ([]*entities.PbsOrtbSeatBid, []error)
+}
+
+type auctionOptions struct {
```

Comment:
How about renaming this struct and the sub structs to something along the lines of:
```
type extraAuctionInfo struct {
	request  extraAuctionReqInfo
	response *extraAuctionResponseInfo
}

type extraAuctionResponseInfo struct {
	mu                      sync.Mutex
	fledge                  *openrtb_ext.Fledge
	bidsFound               bool
	bidderResponseStartTime time.Time
}

// extraAuctionReqInfo holds additional info for bid request execution to maintain clean code and reasonable number of parameters
type extraAuctionReqInfo struct {
	accountDebugAllowed bool
	headerDebugAllowed  bool
	addCallSignHeader   bool
	bidAdjustments      map[string]float64
}
```